tests/ttnn/unit_tests/operations/eltwise/test_fp32.py

```diff
@@ -0,0 +1,24 @@
+# SPDX-FileCopyrightText: © 2023 Tenstorrent Inc.
+
+# SPDX-License-Identifier: Apache-2.0
+
+import pytest
+
+import torch
+
+import ttnn
+from tests.ttnn.utils_for_testing import assert_with_pcc
+
+torch.set_printoptions(precision=10)
+
+with ttnn.manage_device(device_id=0) as device:
+    x_torch = torch.tensor([[0.00001]], dtype=torch.float32)
+    y_torch = -x_torch
+    print("y_torch", y_torch)
+
+    x_tt = ttnn.from_torch(x_torch, dtype=ttnn.float32, layout=ttnn.TILE_LAYOUT, device=device)
+    y_tt = ttnn.neg(x_tt)
+    print("y_tt", y_tt)
+
+    print(x_torch, ttnn.to_torch(x_tt))
+    print(y_torch, ttnn.to_torch(y_tt))
```

Comment:
@KalaivaniMCW , there are no asserts in this test.   Take a look at my comment on https://github.com/tenstorrent/tt-metal/pull/14874

Reply:
Yes it is passing with assert. @eyonland 

<img width="1512" alt="Screenshot 2024-11-08 at 10 29 21 PM" src="https://github.com/user-attachments/assets/e79031a4-70a4-468b-8541-3822c501284a">

Reply:
Please add these asserts into your tests.